ci/docker_build_java.sh

```diff
@@ -26,9 +26,14 @@ mkdir -p /build/java
 arrow_src=/build/java/arrow
 
 pushd /arrow
-  rsync -a header java format integration $arrow_src
+rsync -a header java format integration $arrow_src
```

Comment:
How about adding `--delete` `rsync` option or removing `${arrow_src}` before `rsync`?
If there are some built files with OpenJDK 8, OpenJDK 11 build may be conflicted.

Reply:
OK, will do